packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -55,9 +61,136 @@ export const ConnectedServices = () => {
     setErrorText('Sorry, there was a problem refreshing the recovery key.');
     alertBar.show();
   });
-
   const showMobilePromo = !sortedAndUniqueClients.filter(isMobileDevice).length;
 
+  // TODO: should this view name be the exact same as in old-settings? "settings.clients.disconnect"?
```

Comment:
We have an event property in Amplitude events to distinguish between old and new settings metrics, so we have been using the same event names.  Not sure if that answers your question here.

Reply:
It does, thanks!